assets/data/cards_pt-br.json

```
@@ -482,6 +482,72 @@
         "Conceito",
         "Back-end"
       ]
+    },
+    {
+      "title": "NaN",
+      "description": "A propriedade global NaN é um valor especial que significa Not-A-Number(Não é um número).",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Null",
+      "description": "Null é um literal que representa um valor nulo ou vazio.",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Undefined",
+      "description": "O valor global undefined representa um valor indefinido, ou seja, um valor que não foi definido ou atribuído em uma variável.",
+      "tags": [
+        "Conceito",
+        "Back-end",
+        "Front-end"
+      ]
+    },
+    {
+      "title": "Falsy",
+      "description": "Falsy é um valor que se traduz por falso quando avaliado em um contexto Booleano.",
```

Trouxe uma sugestão pensando que essapode ser um pouco técnica para quem não está familiarizado com o termo "contexto Booleano", veja o que acha:

```suggestion
      "description": "Falsy é um valor que, na programação, é considerado como falso quando usado em situações que precisam de uma resposta verdadeira ou falsa. Alguns exemplos de valores falsy são o número 0, uma string vazia ou o próprio null.",
```